go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/ava-labs/awm-relayer
 go 1.18
 
 require (
-	github.com/ava-labs/avalanchego v1.10.14
-	github.com/ava-labs/subnet-evm v0.5.8
-	github.com/ava-labs/teleporter v0.0.0-20231107202133-eea4d6312c35
+	github.com/ava-labs/avalanche-network-runner v1.7.3-0.20231026153158-2931f8a448d9
+	github.com/ava-labs/avalanchego v1.10.15
+	github.com/ava-labs/subnet-evm v0.5.9
+	github.com/ava-labs/teleporter v0.0.0-20231109205404-3a226fa468c8
```

Comment:
should we create a release for Teleporter version?

Reply:
Yeah, will create a release and update this PR with that version

Reply:
Actually, we haven't published any Teleporter releases yet, and I think we should decide on a formal release process before doing so. Since here it's a direct Golang dependency, I'm ok sticking with commit dependencies for now.